intellij-ide.md

```diff
@@ -185,6 +185,21 @@ your debugging session:
 1. In the Debug panel at the bottom, click the **Full Application Restart button**
 (shown with an icon containing a circular arrow).
 
+## Tips and tricks
+
+Keyboard shortcuts:
+  * JetBrains has a good [Keyboard Shortcuts You Cannot Miss](https://www.jetbrains.com/help/idea/keyboard-shortcuts-you-cannot-miss.html) overview
+  * Customize the shortcuts to your preferences in **Preferences > Keymap**
+
+To navigate quickly in your code:
+  * Switch between recently edited files with `cmd-e`
```

Comment:
Cmd+E is Recently opened files,
Recently Edited Files is Cmd+Shift+E.
And on Linux (and Windows if ever supported) shortcuts are different :)